python/dartpy/dynamics/Joint.cpp

```diff
@@ -255,6 +255,16 @@ void Joint(py::module& m)
             return self->setTransformFromChildBodyNode(T);
           },
           ::py::arg("T"))
+      .def(
+          "getTransformFromParentBodyNode",
+          +[](dart::dynamics::Joint* self) -> const Eigen::Isometry3d& {
```

Comment:
```suggestion
          +[](const dart::dynamics::Joint* self) -> const Eigen::Isometry3d& {
```
It seems we could pass `self` by const pointer since this function doesn't modify `Joint`.

Reply:
Sure. Thank you for the suggestion.